src/commands/utils/produceMessage.test.ts

```diff
@@ -26,29 +26,7 @@ describe("commands/utils/produceMessage.ts createProduceRequestData()", function
     sandbox.restore();
   });
 
-  it("should create request data with 'type' set for CCloud topics", async function () {
-    const result = await createProduceRequestData(
-      {
-        key: "test-key",
-        value: "test-value",
-      },
-      {},
-      true,
-    );
-
-    assert.deepStrictEqual(result, {
-      keyData: {
-        type: "JSON",
-        data: "test-key",
-      },
-      valueData: {
-        type: "JSON",
-        data: "test-value",
-      },
-    });
-  });
-
-  it("should create request data without 'type' for local topics", async function () {
+  it("should nest key/value data under keyData.data and valueData.data", async function () {
```

Comment:
Can we assert that type is _not_ sent?

Reply:
Not really necessary since both paths initially go through `createProduceRequestData()` and we have a number of deep-strict-equal checks in the tests (that were failing earlier in this branch because we still expected `type: undefined` to be returned):
https://github.com/confluentinc/vscode/blob/273b45decb8c1f85f6214053b38c39f60cd2af29/src/commands/utils/produceMessage.test.ts#L15-L18

And then a level deeper in `extractSchemaInfo()` tests to make sure we don't inject anything extra in there either (also previously broken earlier in this branch):
https://github.com/confluentinc/vscode/blob/273b45decb8c1f85f6214053b38c39f60cd2af29/src/commands/utils/produceMessage.test.ts#L67-L71